cwms-data-api/src/main/java/cwms/cda/data/dto/TimeSeries.java

```diff
@@ -43,6 +39,7 @@
 @XmlAccessorOrder(XmlAccessOrder.ALPHABETICAL)
 @JsonPropertyOrder(alphabetic = true)
 @JsonNaming(PropertyNamingStrategies.KebabCaseStrategy.class)
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
What unknown property is getting deserialized?

Reply:
This was on my list to follow up about too

Reply:
This was for versionDateType but I think this can get removed now since my implementation changed. Will remove on next commit